modules/rpc-util.js

```diff
@@ -0,0 +1,40 @@
+const request = require('request');
+const Logger = require('./logger');
+const { Helper } = require('./helper');
+
+class RpcRequest {
+
+    static fetchBalance(provider, accountHash) {
+        var URL = `${provider}/chains/main/blocks/head/context/contracts/${accountHash}/balance`;
+        request(URL, function (error, response, body) {
+            if(error) {
+                Logger.error(`${error}`);
+            }
+            else {
+                try {
+                    var balance = JSON.parse(body);
+                    Logger.info(Helper.formatTez(balance));
```

Comment:
this method should return a promise and logger statement should be there in calling function. There's no point of creating a util if we add logic here. Util should work as call-function & return value format.